components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -240,8 +240,11 @@ def leave_none_unset_int(s):
 INTERNAL_SETTINGS_MAPPING = {
     "omero.qa.feedback":
         ["FEEDBACK_URL", "http://qa.openmicroscopy.org.uk", str, None],
-    "omero.upgrades.url":
-        ["UPGRADES_URL", "http://upgrade.openmicroscopy.org.uk/", str, None],
+    "omero.upgrade-check.enabled":
```

Comment:
Could probably use some discussion here. Either it's web in name and use, or it's everything.